recipes/gnali/meta.yaml

```diff
@@ -33,13 +33,19 @@ requirements:
     - git >=2
   run:
     - python >=3.6
+    - ensembl-vep ==101
+    - samtools
+    - tabix
+    - libcurl
+    - perl-bio-bigfile
     - pybiomart
     - pandas
     - numpy
     - pysam
     - filelock
     - pyyaml
     - py-bgzip
+    - git >=2
```

Comment:
Do you really need git as run time dependecy?

Reply:
Yes, there's a function that clones a repo (without tagged releases)